web/packages/teleport/src/Discover/SelectResource/SelectResource.tsx

```diff
@@ -469,6 +462,23 @@ function makeResourcesWithHasAccessField(acl: Acl): ResourceSpec[] {
   });
 }
 
+/**
+ * getPlatform uses getPlatform from utils and narrows down the result to one of the three platforms
+ * used within SelectResource.
+ */
+function getPlatform(): Platform {
+  const platformType = utilsGetPlatform();
+  if (platformType.isMac) {
+    return UtilsPlatform.PLATFORM_MACINTOSH;
+  }
+  if (platformType.isLinux) {
+    return UtilsPlatform.PLATFORM_LINUX;
+  }
+  if (platformType.isWin) {
+    return UtilsPlatform.PLATFORM_WINDOWS;
+  }
+}
+
```

Comment:
This is temporary. At the time I didn't realize that the other platforms in the original `Platform` enum aren't needed.

In the subsequent PR, I'm going to rename the existing `getPlatform` function to something like `getPlatform__deprecated` and add a new function there which just returns one of these three.